be/src/util/hash_util.hpp

```diff
@@ -132,60 +132,11 @@ class HashUtil {
     // refer to https://github.com/apache/commons-codec/blob/master/src/main/java/org/apache/commons/codec/digest/MurmurHash3.java
     static const uint32_t MURMUR3_32_SEED = 104729;
 
-    ALWAYS_INLINE static uint32_t rotl32(uint32_t x, int8_t r) {
-        return (x << r) | (x >> (32 - r));
-    }
-
-    ALWAYS_INLINE static uint32_t fmix32(uint32_t h) {
-        h ^= h >> 16;
-        h *= 0x85ebca6b;
-        h ^= h >> 13;
-        h *= 0xc2b2ae35;
-        h ^= h >> 16;
-        return h;
-    }
-
     // modify from https://github.com/aappleby/smhasher/blob/master/src/MurmurHash3.cpp
     static uint32_t murmur_hash3_32(const void* key, int32_t len, uint32_t seed) {
```

Comment:
Check the comment: modify from https://github.com/aappleby/smhasher/blob/master/src/MurmurHash3.cpp

Why modify the origin code in the past? Is the modified method the same as murmur_hash3_x86_32?

Reply:
The modified method is the same as murmur_hash3_x86_32, just different in returning value.